src/pages/ReportDetailsPage.tsx

```diff
@@ -312,6 +333,7 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                             )}
                         </View>
                     </View>
+                    {(isGroupChat || isChatRoom) && <ChatDetailsQuickActionsBar report={report} />}
```

Comment:
@ZhenjaHorbach Why we move the position  of `ChatDetailsQuickActionsBar` to above description